src/content/packages-and-plugins/developing-packages.md

```diff
@@ -485,6 +497,21 @@ Use the following instructions to add `HelloPod` with the version `0.0.1`:
 
 The pod should appear in the installation summary.
 
+If your plugin requires a privacy manifest, for example,
+if it uses any **required reason APIs**,
+update the `PrivacyInfo.xcprivacy` file to
+describe your plugin's privacy impact,
+and add the following to the bottom of your podspec file:
+
+```ruby
+s.resource_bundles = {'shared_preferences_foundation_privacy' => ['shared_preferences_foundation/Sources/shared_preferences_foundation/Resources/PrivacyInfo.xcprivacy']}
+   ```
```

Comment:
`shared_preferences_foundation_privacy` is the example from `shared_preferences_foundation`, maybe this should be 
```ruby
s.resource_bundles = {'your_plugin_privacy' => ['your_plugin/Sources/your_plugin/Resources/PrivacyInfo.xcprivacy']}
```
or some appropriate placeholder?